docs/workflows/sign-images.rst

```diff
@@ -298,3 +298,27 @@ It is possible to specify a single manifest identified by tag or a list of manif
 by proviging ``tags_list`` option to the call.
 Note that ``manifest lists`` are not signed, instead all the image manifests that manifest lists
 contain, are signed.
+
+Push signed images to the Pulp Registry
+=======================================
+
+Signatures can be pushed to the Pulp Registry via the extensions API. This API exposes an endpoint
+for reading and writing image signatures.
+
+To add a new signature to an image, issue the following PUT request::
+
+    $ http PUT http://localhost:24817/extensions/v2/<namespace>/<name>/signatures/sha256:<manifest-digest> < signature.json
+
+The JSON payload should have the following structure::
+
+    {
+      "schemaVersion": 2,
+      "type":    "atomic",
+      "name":    "sha256:4028782c08eae4a8c9a28bf661c0a8d1c2fc8e19dbaae2b018b21011197e1484@cddeb7006d914716e2728000746a0b23",
+      "content": "<cryptographic_signature>"
+    }
+
+This step can be also done via podman. After configuring a GPG keyring, it is possible to issue
```

Comment:
... via podman or skopeo, or any another client that supports registry extensions API for signatures.